pages/linux/xset.md

```diff
@@ -0,0 +1,12 @@
+# xset
+
+> User preference utility for X.
```

Comment:
X? Do you mean X11 or the X server?

Reply:
This is copied from the man page.
Isn't X11 just the version of X?
And the X server the actual server implementing the X protocol?
But we can use whatever you want here :)

Reply:
Ah, I see. Let's keep it as-is then.